scripts/system_monitoring_functions/cpu_stats.sh

```
@@ -0,0 +1,58 @@
+get_average_cpu_stats() {
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)